db/db.pl

```diff
@@ -2471,6 +2472,12 @@ sub huntsUpdate
       },
       "notifier": {
         "type": "keyword"
+      },
+      "unrunnable": {
+        "type": "boolean"
+      },
+      "failedSessionIds": {
```

Comment:
Where is that getting used?

Reply:
I asked @31453 to add now so a new db version isn't required for #1512 1512